.eslintrc.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
     es6: true,
     mocha: true,
   },
-  plugins: ['@typescript-eslint', 'prettier'],
+  plugins: ["import", "prettier", "@typescript-eslint"],
```

Comment:
What's the motivation for needing the `import` plugin?

Reply:
This plugin intends to support linting of ES2015+ (ES6+) import/export syntax, and prevent issues with misspelling of file paths and import names. All the goodness that the ES2015+ static module syntax intends to provide, marked up in your editor.

Reply:
Having said that, we could remove this plugin if we don't think it provides value.